modules/textures/src/lib/parsers/parse-ktx.js

```diff
@@ -16,6 +18,15 @@ const KTX2_ID = [
   0x0a
 ];
 
+// TODO Extend mapping vkFormat to WebGl internal formats.
+// Mapping provided here http://github.khronos.org/KTX-Specification/#formatMapping
+// Vulkan format mapping numbers to names: https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VkFormat.html
+const PIXEL_FORMATS = {
+  133: GL.COMPRESSED_RGBA_S3TC_DXT1_EXT,
```

Comment:
I guess we need to expand this table to handle more formats. 

I actually thought these constants were already mapped since this is a Khronos format (same org that standardizes WebGL).

Reply:
Extended